test/cli/dune

```diff
@@ -247,3 +247,102 @@
  (action
   (diff fmterr_stdin_and_name_bad_kind.expected
     fmterr_stdin_and_name_bad_kind.output)))
+
+;; This test's output is ignored because it's not stable
+(rule
+ (targets conf_bad_version.output)
+ (deps
+  (source_tree roots/bad_version))
+ (action
+  (with-outputs-to
+   %{targets}
+   (chdir
+    roots/bad_version
+    (system "! %{bin:ocamlformat} --impl - < %{dep:sample/a.ml} 2>/dev/null")))))
```

Comment:
Note that it is important that the error message is printed on stderr and not stdout, so IIUC redirecting just fd 2 achieves more than just ignoring unstable output, it also checks that the error goes to the right stream.

Reply:
Yes, the test does that. Do you mean I should add that to the comment ?

Reply:
I was just thinking to adjust the comment, to avoid future confusion.

Reply:
Did.